Arrowgene.Ddon.Database/Sql/Core/DdonSqlDbCharacter.cs

```diff
@@ -49,19 +121,24 @@ public bool CreateCharacter(Character character)
             {
                 return false;
             }
-
             character.Id = (uint) autoIncrement;
+
+            StoreCharacterData(character);
```

Comment:
This escalated quickly, I am also unsure myself what is the best way to deal with this hughe amount of fields and properties.
We can see if we want to optimise this in the future, for now I think it is a very good way to show what is required to store.

As for points that I would think of improoving:

- I would like to see individual files for each table, to split it into more managable parts. This class is marked as partial, so it should still be able to access al functionalities from the other files, just keeps related data structures and queries grouped in separate files

- I would like to have basic crud operations for each table, perhaps have Insert or Replace be a separate function called "InsertOrReplaceCharacter" to have a api surface that communicates what the underlying function does.

- I am also thinking of perhap splitting the character table up into 1:1 relations, so that if we update the visuals that is stored in a separate table for each character. It perhaps makes some operations like reading visuals of characterX and apply it to characterY, without supplying all other character data.

- lastly, for character creation or quries that touch multiple tables and depend on each other, we should investigate into transactions, and make these functions that require ALL or FAIL scenarious, to be a transaction, that can be rolled back on failure (https://www.sqlite.org/lang_transaction.html) otherwise we might end up with partial state